packages/presentation/lib/screen/movie_details/model/comments_model.dart

```diff
@@ -0,0 +1,15 @@
+class CommentsModel {
+  const CommentsModel({
+    required this.comment,
+    required this.dateComment,
+    required this.rating,
+    required this.userName,
+    required this.userImage,
+  });
+
+  final String? comment;
+  final String? dateComment;
+  final double? rating;
+  final String? userName;
+  final String? userImage;
```

Comment:
is this ui model? u can exclude null by making check in mapper if it exists. on my opinion it is better solution instead make null checks on ui, but up to u